package.json

```diff
@@ -13,7 +13,7 @@
     "relay2ts": "relay2ts src/components/*.tsx --update",
     "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/master && yarn install && npm run dump-schema -- ../../data",
     "sync-colors": "cd externals/elan && git fetch && git checkout origin/master && cp components/lib/variables/colors.json ../../data",
-    "lint": "tslint -c tslint.json --project tsconfig.json",
+    "lint": "tslint -c tslint.json --project tsconfig.json --fix",
```

Comment:
> I don’t think we should do this, it would also auto-fix fails on CI (but not commit them) and be pseudo-green.

What status does the `tslint` command return if auto-fix is on and there's a failure? Rubocop always returns 0 if auto-fix is on, but that might not be the case in tslint.